frontend/src/component/layout/MainLayout/NavigationSidebar/NewInUnleash/NewInUnleashDialog.tsx

```diff
@@ -46,13 +46,14 @@ const StyledMainTitle = styled('div')(({ theme }) => ({
     lineHeight: 1.2,
 }));
 
-const StyledLink = styled(Link<typeof RouterLink | 'a'>)(({ theme }) => ({
+const StyledLink = styled(Link)(({ theme }) => ({
```

Comment:
This is the same simplification done in https://github.com/Unleash/unleash/pull/11022. There's no real point in the extra indirection when we can use `Link` directly.